### ISSUE TITLE
Change project path

### ISSUE THREAD

**Issue by ciyer** [SwissDataScienceCenter/renku#692]
# Description

Users may want to change the path of a project, for example, to transfer ownership of a project from an individual user to a group or from one group to another. 

GitLab currently prevents this because we use the docker image repository and there are registry tags associated with the project. Changing the project path would also have impact on the KG.

We need to develop a solution to enable this.

**Comment by rokroskar** [comment]
Are you suggesting we automatically purge the images for the user? Or provide the docs on how to do this?

**Comment by ciyer** [comment]
Yes, I'm suggesting we inform the user that changing the path of a project has these consequences and, if approved, do things like purge images for the user.

**Comment by gavin-k-lee** [comment]
Related to #860
